app/lib/validation-helpers.js

```diff
@@ -0,0 +1,32 @@
+const validNewMicrochip = /^\d+$/
+
+const FIFTEEN = 15
+
+const validateMicrochip = (value, helpers, compareOrig = false) => {
+  let elemName = helpers.state.path[0]
+
+  // Compare new value against original to determine if already pre-populated in the DB
```

Comment:
We probably don;t need lines 8 - 17 as this was to handle old format microchips. As the Police are entering new format ones, then should never need it. I suppose no harm in leaving in though, as long as it's covered by a test

Reply:
there is a test for this.  we can remove later as yes, guess not needed, but want to reduce work as we have 1 hr.